ui2/src/pages/clearing/Members.tsx

```diff
@@ -115,6 +131,51 @@ const ClearingMembersComponent: React.FC<RouteComponentProps & ServicePageProps<
           };
         })}
       />
+      <Header as="h2">Cleared Trades</Header>
+      <StripedTable
+        loading={clearedTradesLoading}
+        rowsClickable
+        headings={['Provider', 'Match Id', 'Maker Order Id', 'Taker Order Id', 'Quantity', 'Price']}
+        rows={clearedTrades.map(c => {
+          return {
+            elements: [
+              getName(c.payload.provider),
+              c.payload.execution.matchId,
+              c.payload.execution.makerOrderId,
+              c.payload.execution.takerOrderId,
+              c.payload.execution.quantity,
+              c.payload.execution.price,
+              <Button
+                size="small"
+                className="ghost"
+                variant="contained"
+                onClick={() => handleNovation(c)}
+              >
+                Novate Trade
+              </Button>,
+            ],
+          };
+        })}
+      />
+      <Header as="h2">Cleared Trade Sides</Header>
+      <StripedTable
+        loading={clearedTradeSidesLoading}
+        rowsClickable
```

Comment:
ex: here `rowsClickable` works because there are no buttons in the row data, and the onClick function will apply to the whole row